Sources/Jetpack/Info.plist

```diff
@@ -485,17 +436,32 @@
 				</array>
 				<key>UIPrerenderedIcon</key>
 				<true/>
+				<key>WPRequiresBorder</key>
+				<true/>
 			</dict>
-			<key>Spectrum on Black</key>
+			<key>Stroke Dark</key>
 			<dict>
 				<key>CFBundleIconFiles</key>
 				<array>
-					<string>spectrum-on-black-icon-app-60</string>
-					<string>spectrum-on-black-icon-app-76</string>
-					<string>spectrum-on-black-icon-app-83.5</string>
+					<string>stroke-dark-icon-app-60</string>
```

Comment:
Xcode automatically re-ordered everything when I changed "Launch Screen" to "LaunchScreen" to remove spaces from file names.